requirements.txt

```diff
@@ -5,12 +5,15 @@ sqlalchemy >= 1.3
 https://github.com/lsst/sphgeom/archive/master.tar.gz
 
 # optional
+backoff >= 1.10
 boto3 >= 1.13
+botocore >= 1.15
 moto >= 1.3
 pandas >= 1.0
 numpy >= 1.17
 matplotlib >= 3.0.3
 pyarrow >= 0.16
+urllib3 >= 1.25
```

Comment:
I think urllib3 is part of python core so I'm not sure this is needed is it?

Reply:
It's not. I think it's a dependency of boto3 so we had it in the build before, which is why we were getting it in stack traces.

Reply:
Oh yes. Sorry about that.